manifest.json

```diff
@@ -24,10 +24,8 @@
     "webRequest",
     "webRequestBlocking",
     "storage",
-    "*://msdn.microsoft.com/*", 
-    "*://*.msdn.microsoft.com/*",
-    "*://docs.microsoft.com/*",
-    "*://*.docs.microsoft.com/*",
+    "*://learn.microsoft.com/*", 
```

Comment:
Would probably just make a new item, just in case MS gets any ideas, same as I'd keep the other entries. Guess this is such a small performance impact, it depends on AirLancer's philosophy here.

Reply:
Oh nevermind I just saw the other one went in exactly the other direction😅

Reply:
Well, docs.microsoft.com does a `HTTP 301 Permanently moved` redirect. I guess we should follow suite.